emscripten/npm/src/verovio-toolkit.js

```diff
@@ -119,10 +123,10 @@ export class VerovioToolkit {
         }
         var dataArray = new Uint8Array(data);
         var dataSize = dataArray.length * dataArray.BYTES_PER_ELEMENT;
-        var dataPtr = Module._malloc(dataSize);
-        Module.HEAPU8.set(dataArray, dataPtr);
+        var dataPtr = this.module._malloc(dataSize);
```

Comment:
can't this just be `this.VerovioModule` directly? (also, are you missing the method call parens `()`?)

Reply:
See my comment below. My syntax is correct. Class getters are accessed as properties in JavaScript and do not need to be called: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get